adapter/slack.py

```diff
@@ -155,68 +284,77 @@ async def list_bots(self, request: Request):
             return Response(status_code=200)
         except SlackApiError as e:
             raise HTTPException(status_code=400, detail=f"Slack API Error : {e}")
-    
+
     def event_message(self, event):
-        if 'thread_ts' in event:
+        if "thread_ts" in event:
             self.event_message_replied(event)
-    
+
     def event_message_replied(self, event):
         parent_message = self.app.client.conversations_history(
-            channel=event['channel'],
+            channel=event["channel"],
             inclusive=True,
-            oldest=event['thread_ts'],
+            oldest=event["thread_ts"],
             limit=1,
-            include_all_metadata=True
+            include_all_metadata=True,
         )
-        
-        parent_user = parent_message['messages'][0]['user']
-        
-        if parent_user == self.app_user_id and parent_message['messages'][0].get("metadata") != None:
+
+        parent_user = parent_message["messages"][0]["user"]
+
+        if (
+            parent_user == self.app_user_id
+            and parent_message["messages"][0].get("metadata") != None
+        ):
             self.bot_replied(event, parent_message)
-        
+
     def bot_replied(self, event, parent_message):
-        bot_slug = parent_message["messages"][0]["metadata"]["event_payload"]["bot_slug"]
-        question = event['text']
-        thread_ts = event['thread_ts']
+        bot_slug = parent_message["messages"][0]["metadata"]["event_payload"][
+            "bot_slug"
+        ]

```

Comment:
Why is the "bot_slug" separated into another line? Is it because the line is too long?

Reply:
Yes. That's part of the autoformatter